docs/01-installing.md

```diff
@@ -1,3 +1,9 @@
+# Prerequisites
+
+* There were some breaking changes in the development. To rule out incompatibilities make sure you have installed the latest version of [``Noevim 0.5``](https://github.com/neovim/neovim/releases/tag/v0.5.0).
+* Have `npm` and `node` installed on your system
```

Comment:
If we're going to ask for `npm` and `node` we might as well also ask for `cargo` since the rust dependencies can't get installed without it, and telescope won't work without the rust dependencies.

Reply:
Is python also needed?

> If we're going to ask for `npm` and `node` we might as well also ask for `cargo` since the rust dependencies can't get installed without it, and telescope won't work without the rust dependencies.

Is python also needed? In this case I should add it 2

Reply:
Done, please check